datafusion/core/src/datasource/file_format/write/mod.rs

```diff
@@ -149,15 +145,14 @@ impl<W: AsyncWrite + Unpin + Send> AsyncWrite for AbortableWrite<W> {
 
 /// A trait that defines the methods required for a RecordBatch serializer.
 #[async_trait]
-pub trait BatchSerializer: Unpin + Send {
+pub trait SerializationSchema: Sync + Send {
```

Comment:
What is the rationale for renaming this trait? It doesn't seem directly related to a `Schema` 🤔 I think the original name `BatchSerializer` better matches what the trait does